package.json

```diff
@@ -35,6 +35,7 @@
     "babel-jest": "^26.6.3",
     "eslint": "^7.29.0",
     "eslint-config-next": "^11.0.0",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
```

Comment:
@coderitual IMO it would be nice to use something like this in `brainly-frontend` repo to point out some basic a11y issues

Reply:
Yep agree. My only concern is related to the rule set we need to require. Wrong a11y attrs are worse than no attrs.

also we need someone who will align the app to follow new es lint rules. There is no point having es lint rules as a warning ( we need to require following rules) and we cannot require following rules before app is aligned. 

Introducing new eslint rule is always a bit painful and we need dedicated developer to make that happen since it is gonna be a lot of work.

Personally I would see maybe even more value having a11y plugin for storybook in the app and es lint later

Reply:
@coderitual yup, storybook would be a great start!